src/elements/JobsPanelTable/JobsPanelTableView.js

```diff
@@ -45,9 +49,13 @@ const JobsPanelTableView = ({
         ) {
           return section === 'data-inputs' ? (
             <EditableDataInputsRow
+              comboboxMatchesList={comboboxMatchesList}
```

Comment:
here, don't rename prop, but change to : 
```suggestion
              comboboxMatchesList={sectionData.comoboxoMatchesList}
```